distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+==== Certificate Management
+
+${branding} uses certificates in two ways:
+
+["lowerroman"]
+. Transmitting and receiving encrypted messages.
+. Authenticating an incoming user request.
+
+For more certificate configuration options, see <<_configuring_${ddf-security}, Configuring ${ddf-security}>>.
+
+===== Keystores
+
+To manage certificates and keys, ${branding} includes a default keystore.
```

Comment:
Lines 14-45 need to be redone. While we're at it, lets' delete all the way to 51. Here is how I'd like it to look:

----

Certificates (and certificates with keys) can be managed in the Admin UI Console. Navigate to the Security application and select it. After it opens, select the notebook tab labeled "Certificates". This view shows the alias (name) of every certificate in the trust store and the key store. It also displays if the entry includes a private key ("Is Key") and the encryption scheme (typically "RSA" or "EC"). 

This view allows administrators remove certificates from  ${branding}'s key and trust stores. It also allows administrators to import certificates and private keys into the keystores with the "+" button. The import function has two option: import from a file or import over HTTPS. The file option excepts a Java Keystore file or a PKCS 12 key store file.  Because keystores can hold many keys, the import dialog asks the administrator to provide the alias of the key to import. Private keys are typically encrypted and the import dialog prompts the administrator to enter the password for the private key. Additionally, keystore files themselves are typically encrypted and the dialog asks for the keystore ("Store")  password.

The name and location of the ${branding} trust and key stores can be changed by editing the system properties files, etc/system.properties. Additionally, the password that ${branding} uses to decrypt (unlock) the key and trust stores can be changed here. 

IMPORTANT: ${branding} assumes that password used to unlock the keystore is the same password that unlocks private keys in the keystore.

```
# Set the keystore and truststore Java properties
javax.net.ssl.keyStore=etc/keystores/serverKeystore.jks
javax.net.ssl.keyStorePassword=changeit
javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
javax.net.ssl.trustStorePassword=changeit
javax.net.ssl.keyStoreType=jks
javax.net.ssl.trustStoreType=jks
```
Lastly, ${branding} supports two keystore formats: Java Keystore (JKS) and PKCS12. The default keystores are JKS.